tests/jenkins/nrnivmodl.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+. /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
+module load neuron/2018-10/python3/parallel
```

Comment:
same here

Reply:
I used this particular version because `module load neuron` loads by default `neuron/2018-10/python3/serial` and running `nrnivmodl` of this version to build the `special` for the `testcorenrn` fails with the following error:
```
libtool: link: unable to infer tagged configuration
libtool: link: specify a tag with `--tag'
```

Reply:
ok..understood. for neuron you can specify full module then.